src/App.jsx

```diff
@@ -1,35 +1,35 @@
-import { useState } from 'react'
+import {useState} from 'react'
```

Comment:
These first three imports are things you aren't using. Let's remove them to keep everything clean.